Podfile.lock

```diff
@@ -7,15 +7,25 @@ PODS:
   - StencilSwiftKit (2.5.0):
     - Stencil (~> 0.11)
   - SwiftLint (0.25.1)
-  - Yams (0.7.0)
+  - Yams (1.0.0)
```

Comment:
🤔 what's new in Yams 1.0.0? Maybe we could use some new stuff…

Reply:
> Rename ScalarRepresentableCustomizedForCodable to YAMLEncodable.
And some new documentation available at https://www.jpsim.com/Yams/.

Reply:
So nothing for us really, I think. It's just nice that we now have a 1.0 version we can depend on.

Reply:
👍